MapboxCoreNavigation/RouteController.swift

```diff
@@ -221,13 +221,7 @@ open class RouteController: NSObject, Router {
      Monitors the user's course to see if it is consistantly moving away from what we expect the course to be at a given point.
      */
     func userCourseIsOnRoute(_ location: CLLocation) -> Bool {
-        // if we have yet to travel along the current leg, don't check for heading conformance
-        guard routeProgress.currentLegProgress.distanceTraveled > 0 else {
-            movementsAwayFromRoute = 0
-            return true
-        }
-        
-        let nearByCoordinates = routeProgress.currentLegProgress.nearbyCoordinates
+        let nearByCoordinates = routeProgress.nearbyCoordinates
```

Comment:
Does it make sense to keep the currentLegProgress.nearbyCoordinates in addition to the new routeProgress.nearbyCoordinates? should we be deprecating the older one?

also, the B in nearby should be a small b, but that's ultra-minor

Reply:
I've no real feeling on the subject either way, but @1ec5 did argue for keeping it the other day, and his rationale (which I cannot remember OOTOMH) made sense to me.